src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx

```diff
@@ -29,6 +32,14 @@ export const IncidentsDetail = ({
   const { _display, text, location, extra_properties, _links } = incidentsDetail
   const attachments = _links?.['sia:attachments']
 
+  const attachmentsUser = attachments?.filter(
+    (attachment) => !attachment.created_by
+  )
+
+  const attachmentsMunicipality = attachments?.filter(
+    (attachment) => attachment.created_by
+  )
+
```

Comment:
is `created_by` the best way to distinguish between these two? Attachments added by the user never have a created_at?

Reply:
yep, its distinguishing the two by the user and municipality